.circleci/config.yml

```diff
@@ -17,7 +18,7 @@ jobs:
                     - checkout
                     - node/install:
                         install-yarn: false
-                        node-version: latest
+                        #node-version: latest
```

Comment:
?

Reply:
I had to comment this out because the newer Orb node: circleci/node@4.7.0 doesn't accept the version 'latest', (however I think if you don't specify the node-version it does install the latest version)

Reply:
CheckUrlExists for HTTPS urls returns false.

Reply:
removed comment and added fix to index.js for https.